### ISSUE TITLE
menuBar has accessibility violation

### ISSUE THREAD

**Issue by sh977218** [primefaces/primeng#14680]
### Describe the bug

I'm using `axe-core` (https://www.npmjs.com/package/axe-core) library to check a11y of DOM and it gives a violation error about primeNG's menuBar violation:
aria-allowed-attr
Fix all of the following:  ARIA attribute is not allowed: aria-level="1"

The error says: `role="menuitem"` is not compatible with `aria-level`.

Is this a bug in `axe-coire` or primeNG?

### Environment

Running `axe-core` check on html DOM, using Playwright.

### Angular version

16

### PrimeNG version

16.9.4-lts

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18

### Browser(s)

Chrome
